sheerwater_benchmarking/baselines/__init__.py

```diff
@@ -1,11 +1,10 @@
 """Baseline models for the Sheerwater benchmarking project."""
-from .climatology import climatology
-from .ecmwf import iri_ecmwf, ecmwf_agg, ecmwf_rolled
+from .climatology import climatology, climatology_standard_30yr
+from .ecmwf import ecmwf_agg
 
 # Use __all__ to define what is part of the public API.
 __all__ = [
     climatology,
-    iri_ecmwf,
```

Comment:
I've made a bunch of these intermediate functions non-public. They're not all in base180 and really don't represent polished products. What do you think, @adkinsjd ?